rust/connlib/tunnel/src/peer.rs

```diff
@@ -142,21 +134,13 @@ where
         let mut buf = [0u8; MAX_SCRATCH_SPACE];
 
         let packet = match self.tunnel.lock().update_timers(&mut buf) {
-            TunnResult::Done => return Ok(()),
+            TunnResult::Done => return Ok(Bytes::new()),
```

Comment:
I'd rather return a `Result<Option<Bytes>>` so that we don't send empty packets.

We can also check if the packet is empty and not send it.

Reply:
This is because afaik in `webrtc` empty packets are a thing, so something could be sent to the reciever.